project-code/app/com/edulify/modules/geolocation/GeolocationPlugin.java

```diff
@@ -1,39 +0,0 @@
-package com.edulify.modules.geolocation;
-
-import com.edulify.modules.geolocation.providers.FreegeoipProvider;
-import play.Application;
-
-public class GeolocationPlugin extends play.Plugin {
```

Comment:
I don't think we should remove the plugin class. We could just deprecated it, like Play did, and let users know that it will be remove soon.